src/aldrich/TrouserPattern.java

```diff
@@ -236,7 +238,7 @@ protected void createBlocks() {
         backblock.addKeypoint(new Vector2D((d_BodyRise / 2.0), (-(((b_Hips / 12.0) + 2) - ((((b_Hips / 12.0)) + 2.0) / 4.0)))));
 
         // Adding Step 24
-        backblock.addKeypoint(new Vector2D((d_BodyRise + 0.5), (-((((b_Hips / 12) + 2.0) + ((b_Hips / 16.0) + 1.0) + ((b_Hips / 16.0) + 1.0)) / 2.0 + 0.8))));
+        backblock.addKeypoint(new Vector2D((d_BodyRise + 0.5), -((b_Hips/12)+2+(b_Hips/16)+1+0.8+(((b_Hips/16)+1)/2))));
```

Comment:
All values are double precision so you should add the .0 after values. This is not just for appearances, it also makes life easier for the compiler and avoids unnecessary casting between data types.